docs/contracts.rst

```diff
@@ -85,9 +109,90 @@ To run this example, you will need to install a few extra features:
 
     # Display the new greeting value
     print('Updated contract greeting: {}'.format(
-        greeter.functions.greet().call()
+      greeter.functions.greet().call()
     ))
 
+    # When issuing a lot of reads, try this more concise reader:
+    reader = ConciseContract(greeter)
+    assert reader.greet() == "Nihao"
+
+    .. testsetup::
```

Comment:
This should be `.. doctest::` so that users will be able to see it. `.. testsetup::` blocks are not shown in the docs. There should be no whitespace in front of the dots.